pkgs/applications/science/logic/coq/default.nix

```diff
@@ -68,11 +70,12 @@ let
   '' else "";
   ocamlPackages = if !isNull customOCamlPackages then customOCamlPackages
     else with versions; switch coq-version [
-      { case = range "8.14" "8.14"; out = ocamlPackages_4_12; }
+      { case = range "8.16" "8.17"; out = ocamlPackages_4_14; }
+      { case = range "8.14" "8.15"; out = ocamlPackages_4_12; }
       { case = range "8.11" "8.13"; out = ocamlPackages_4_10; }
```

Comment:
FWIW, Coq 8.11 does support OCaml 4.10 (actually even OCaml 4.11), but starting at version 8.11.1 only (according to the version constraints in the opam-repository files).

Again, according to the same opam-repository constraints, support for OCaml 4.14 starts at Coq 8.13.0.

Reply:
Thanks for the feed-back. I guess such a change can wait a further PR.